core/trino-main/src/main/java/io/trino/execution/QueryManagerConfig.java

```diff
@@ -284,6 +285,20 @@ public QueryManagerConfig setQueryMaxExecutionTime(Duration queryMaxExecutionTim
         return this;
     }
 
+    @NotNull
+    @MinDuration("1ns")
```

Comment:
nit: why not 0? Duration itself can be `0`.

Reply:
Removed the constraint to be consistent with `max-execution-time`